### ISSUE TITLE
Conveying general ASGI compatibility?

### ISSUE THREAD

**Issue by florimondmanca** [fullonic/brotli-asgi#2]
Hi,

Congrats again for pulling this off :-) I really appreciate how this adds an interesting additional piece that will benefit to the entire ASGI ecosystem.

Sort of related though, from what I can see this package is currently presented as "Brotli built with/for Starlette". Currently I find like the language and copy might give the impression that this middleware can only be used with Starlette-based frameworks — when in fact it actually works for _any_ ASGI app (so even eg Django 3, Quart, Sanic, etc).

It's certainly easier to build a middleware like this using Starlette to benefit from its datastructures such as `Headers`, but as I found out while working on [msgpack-asgi](https://github.com/florimondmanca/msgpack-asgi) it's actually possible to treat (and present) that as an implementation detail, and market the library as "XYZ for ASGI apps".

Let me know what you think; I can submit a PR for a README update if this sounds interesting to you. :-) (I'm also thinking a name like `brotli-asgi` or `asgi-brotli` would help convey this "it works for any ASGI app/framework" idea even better - interested in hearing your thoughts on this.)

**Comment by fullonic** [comment]
When I did the PR to your great collection [awesome-asgi](https://github.com/florimondmanca/awesome-asgi) I saw your package. Nice one!! I look forward to used in my next project. :)
I wasn't sure at the beginning if it would work with others frameworks, but if you think so, I'm happy to accept your README update. And thanks for do it.
About changing the name, sounds good to me. It makes it clear and also follow the same naming style than `msgpack-asgi`, `timing-asgi`, etc.. I will do it during the weekend.
Thanks again for your help :+1:
